class-work/02-deploy-python.qmd

```diff
@@ -62,21 +62,25 @@ vetiver.model_card()
 ```{python}
 from vetiver import VetiverAPI
 
-api = __
+api = ___
 api.run()
 ```
 
 ## Deploy to Posit Connect 🚀
 
 ```{python}
 from rsconnect.api import RSConnectServer
+import os
```

Comment:
I intentionally omitted the creation of these variables from the slide companion, but can add them in to slides if desired!